omod/src/main/webapp/fragments/labQueueList.gsp

```diff
@@ -90,6 +90,9 @@
     background-color: #FFF;
     border: 1px solid #dddddd;
 }
+.fade:not(.show) {
```

Comment:
@slubwama why are you hardcoding this style here, should it not be global?

Reply:
@ssmusoke when put in global, not sure how it will affect the entire application.